lms/djangoapps/instructor_task/tasks_helper/misc.py

```diff
@@ -140,6 +141,27 @@ def upload_proctored_exam_results_report(_xmodule_instance_args, _entry_id, cour
     return task_progress.update_task_state(extra_meta=current_step)
 
 
+def _get_csv_file_content(csv_file):
+    """
+    returns appropriate csv file content based on input and output is
+    compatible with python versions
+    """
+    if (not isinstance(csv_file, str)) and six.PY3:
```

Comment:
type of `csv_file` will be a `file` object or `str` content of the file itself, checking against `isinstance(csv_file, file)` (file type) is not recommended so the condition negating the `str` type.